tests/integration/page/page_enhance_within.js

```diff
@@ -0,0 +1,21 @@
+asyncTest( "Option enhanceWithin", function() {
+	$.testHelper.pageSequence([
+		function() {
+			$( "#open-enhance-within" ).click();
+		},
+		function() {
+			deepEqual( $( "#enhance-within" ).find( ":mobile-toolbar" ).length, 1,
+				"Page with option enhanceWithin contains a toolbar widget" );
+			$.mobile.back();
+		},
+		function() {
+			$( "#open-no-enhance-within" ).click();
+		},
+		function() {
+			deepEqual( $( "#no-enhance-within" ).find( ":mobile-toolbar" ).length, 0,
```

Comment:
Again, I don't understand. I'm looking for a certain element inside the page with ID ```no-enhance-within```, an element which is an instance of a toolbar widget, and I'm hoping not to find any such element.